src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -140,39 +117,32 @@ public static Logger getLogger() {
 
   public static void gameOverClearCache() {
     // Is static, set to null so that we don't keep the data around after a game is exited
-    s_battleCalculator.setGameData(null);
+    ProBattleUtils.setData(null);
```

Comment:
Code smells:
- null values, use `Optional.of()` instead
- passing null, instead have a static method in `ProBattleUtils` called "clearCache()", and have that call `setData(null)`, and hopefully the setData() method can be made static or removed entirely.
- programming by side effects and tight coupling. Can be solved with a "clearCache" event listener